src/typing_extensions.py

```diff
@@ -2624,12 +2658,67 @@ def __new__(cls, typename, bases, ns):
                 nm_tpl.__init_subclass__()
             return nm_tpl
 
-    def NamedTuple(__typename, __fields=None, **kwargs):
-        if __fields is None:
-            __fields = kwargs.items()
+    def NamedTuple(__typename, __fields=_sentinel, **kwargs):
+        """Typed version of namedtuple.
```

Comment:
If we're adding a docstring the normal way here, we no longer need to monkey-patch it onto the function afterwards:

https://github.com/python/typing_extensions/blob/f4adac5cb359f74ccbab47a36195f4d5e69a558a/src/typing_extensions.py#L2726

Reply:
Oh I missed that. I think it's better if we present the same docstring across all versions, so I'll remove the monkey-patching.